### PR TITLE
Create test for base html

### PR DESCRIPTION
**Pull request by Pawelooo**

### CODE REVIEW

**Comment by jacoor** [comment]
@Pawelooo where are we with this?

**Comment by Pawelooo** [comment]
> @Pawelooo where are we with this?

@jacoor  I don't know how to write these tests so that they pass

**Comment by jacoor** [comment]
Ok. So why did you not ask on Discord? That's why we are a team.

File: articles/tests/test_base_page.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import django
+import pytest
+from django.test import Client
+from django.urls import reverse
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "reddit.settings")
+django.setup()
+
+@pytest.mark.django_db()
```
Comment (jacoor) [inline articles/tests/test_base_page.py:7]: this is not a function. Remove ()

File: articles/tests/test_base_page.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import os
+
+import django
+import pytest
+from django.test import Client
+from django.urls import reverse
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "reddit.settings")
```
Comment (jacoor) [inline articles/tests/test_base_page.py]: you havev this in pytest.ini. Remove.

File: articles/tests/test_base_page.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+django.setup()
```
Comment (jacoor) [inline articles/tests/test_base_page.py]: not needed.

**Review by jacoor — CHANGES_REQUESTED** [review]

**Comment by jacoor** [comment]
Found it. 
please add "pytest-django" package to the project and remove articles/tests.py file.

**Comment by Pawelooo** [comment]
image
I pass these tests

**Comment by jacoor** [comment]
And? Are you asking for something or stating something?
The tests here are not passing and this is the source of truth. 
Have you checked the failing tests here? There is clear into what is wrong.

**Review by jacoor — APPROVED** [review]

**Pull request merged**